src/arch/xen/event.c

```diff
@@ -29,42 +28,6 @@ void hypervisor_callback(void);
 void failsafe_callback(void);
 
 
-static evtchn_handler_t handlers[NUM_CHANNELS];
```

Comment:
This is nice too. Having only this code made a feel of a lot of duplicates. There is still some commented code (`init_events` in kernel_init.c), redundant `xen_shared_info` in traps.c. But that's all that I can easily spot. This change is a big step forward.